repositories/src/main/scala/cromwell/pipeline/datastorage/dto/formatters/UserFormatters.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.pipeline.datastorage.dto.formatters
+
+import cromwell.pipeline.datastorage.dto.{User, UserId}
+import play.api.libs.json.{Json, OFormat}
```

Comment:
just use formatter, checkout option: formatter config would be in our project, but not default one

Reply:
scalafmt has already been used. Turned it on again, and restarted IDE

Reply:
contact me, I will check your configuration